src/components/organisms/ChatSection/ChatSection.tsx

```diff
@@ -1,9 +1,42 @@
 import { Button, Box, Typography, TextField } from "@mui/material";
 import { Send as SendIcon } from "@mui/icons-material";
-import { SentMessage } from "components";
 import styles from "./ChatSection.module.css";
+import { SentMessagesList } from "components";
+import React, { useState } from "react";
+const tempMessage1 = {
```

Comment:
Może coś w tym stylu?
```suggestion
const tempMessage1 : Message= {
```